content/kubermatic/master/tutorials_howtos/external_clusters/aks/_index.en.md

```diff
@@ -5,8 +5,63 @@ weight = 7
 
 +++
 
+## Add AKS cluster
 
+You can add an existing Kubernetes cluster and then manage it using KKP. From the Clusters page, click `External Clusters`.
+Click `Add External Cluster` button and Pick `Azure Kubernetes Cluster` provider.
 
+![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
 
+Select preset with valid credentials or enter AKS `Tenant ID`, `Subscription ID`, `Client ID` and  `Client Secret`, to connect to the provider.
+The credentials should have enough access like Read, Write the cluster and get admin credentials in order to fetch Kubeconfig using api.
 
+![AKS credentials](/img/kubermatic/master/tutorials/external_clusters/aks_credentials.png "AKS credentials")
 
+You should see the list of all available clusters. Select the one and click the `Import Cluster` button. Once imported cluster
+is unique for the project. The same cluster can be added from the other projects.
```

Comment:
s/Once imported cluster is unique for the project/Clusters can be imported only once in single project